app/views/Dashboard/Overview/RegionalBreakdownCard/index.tsx

```diff
@@ -0,0 +1,165 @@
+}
+
+function RegionalBreakdownLabel(props: RegionalBreakdownLabelProps) {
+    const {
+        id,
+        country,
+        color,
+    } = props;
+
+    return (
+        <div
+            className={styles.breakdownLabelWrapper}
+            key={id}
+        >
+            <div
+                style={{
+                    backgroundColor: color,
+                    width: 10,
+                    height: 10,
+                }}
+            />
+            <div className={styles.labelName}>
+                {country}
+            </div>
+        </div>
+    );
+}
+
+function RegionalBreakdownCard(props: RegionalBreakdownCardProps) {
+    const {
+        className,
+    } = props;
+
+    const pieChartInfoRendererParams = useCallback(
+        (_: string, data: PieChartInfoRendererProps) => ({
+            id: data.id,
+            country: data.country,
+            regionalData: data?.regionalData,
+        }), [],
+    );
+
+    const regionalBreakdownLabelParams = useCallback(
+        (_: string, data: RegionalBreakdownLabelProps) => ({
+            id: data.id,
+            color: data.color,
+            country: data.country,
+        }), [],
+    );
+
+    return (
+        <div className={_cs(className, styles.cardInfo)}>
+            <ContainerCard
+                className={_cs(styles.globalStatCard)}
+                contentClassName={styles.responsiveContent}
+                heading="Total number of cases"
+                headingSize="extraSmall"
+                headerDescription="Loreum epsum epsum 2020"
+            >
+                <ResponsiveContainer className={styles.responsiveContainer}>
+                    <BarChart
+                        data={totalCasesBarChart}
+                        barSize={45}
+                    >
+                        <Tooltip
+                            isAnimationActive={false}
+                            allowEscapeViewBox={{
+                                x: false,
+                                y: false,
+                            }}
+                        />
+                        <XAxis
+                            dataKey="name"
+                            tickLine={false}
+                            axisLine={false}
+                        >
+                            <LabelList
+                                dataKey="name"
+                                position="bottom"
+                                fontSize="10"
+                            />
+                        </XAxis>
+                        <Bar
+                            dataKey="amt"
+                            isAnimationActive={false}
+                        >
+                            {totalCasesBarChart?.map((entry) => (
+                                <Cell
+                                    key={`Cell -${entry.id}`}
+                                    fill={COLORS[entry.id % COLORS.length]}
+                                />
+                            ))}
```

Comment:
use list

Reply:
This cannot be implemented as the PieChart component of re-charts does not accept foreign div components

Reply:
This cannot be implemented as the PieChart component of re-charts does not accept foreign div components